pan3d/dataset_builder.py

```diff
@@ -70,6 +71,12 @@ def ctrl(self):
     def data_array(self):
         return self.algorithm.sliced_data_array
 
+    @property
+    def pv_mesh(self):
```

Comment:
is pv_mesh a ParaView proxy, or pyvista mesh or a vtk dataset?

Reply:
I might prefer a `vtk_mesh` name

Reply:
The object returned from this is a `pyvista.core.grid.RectilinearGrid`

Reply:
then name it `rectilinear_grid`

Reply:
Apologies, it could also be a `pyvista.core.grid.StructuredGrid`, according to this logic in `pyvista-xarray`: https://github.com/pyvista/pyvista-xarray/blob/ff112c0979147a085cffe2e82790c039f829e402/pvxarray/accessor.py#L74. Maybe the name could be `pv_grid` or `pyvista_grid`?

Reply:
I would go with either `grid`, `mesh`, `vtk`.

Reply:
Ok, I like the thought of still calling it a mesh, since that's what we call it internally. I changed the name of that property to just "mesh" and added an underscore to all previous instances of "self.mesh": [75bd691](https://github.com/Kitware/pan3d/pull/37/commits/75bd691633ff50bc43447f25c5c53d2475e6cdcd)